.vscode/settings.json

```diff
@@ -4,7 +4,6 @@
     ],
     "VitestRunner.executionArg": "npx --prefix frontend",
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true,
```

Comment:
🤔 i thought this worked correctly and made sure you can run your pytest using vscode

Reply:
it wassss, but it was also causing variables at top level to get evaluated, (and there are two variables which call routes so it was making random route calls everytime i saved which was bugging me)